### PR TITLE
Cloud tables > merge after cloud-databases PR

### PR DESCRIPTION
**Pull request by ljsinclair**
This has the following:
* new /cloud/cloud-tables folder
* new content in there
* new include files under /_includes/cloud
* new include files under /_includes/concepts
FYI, something's gone wrong with the ec2 setup (probably because of fiddling by yours truly) and I can't build these to test. 
* deleted /cloud-ingestion/tables

### CODE REVIEW

**Comment by gthrone25** [comment]
Looks like some of the includes are missing. In that case I'll review the raw files, but these will need to be built locally to ensure they work after the db stuff goes in.

```
  Liquid Exception: Could not locate the included file '/cloud/trial-account-limits.md' in any of ["/Users/gregorythrone/Documents/Molecula/GitPull/documentation/_includes", "/Library/Ruby/Gems/2.6.0/gems/jekyll-theme-primer-0.6.0/_includes"]. Ensure it exists in one of those directories and is not a symlink as those are not allowed in safe mode. in /Users/gregorythrone/Documents/Molecula/GitPull/documentation/cloud/cloud-tables/_posts/2022-11-18-cloud-table-manage.md
jekyll 3.9.0 | Error:  Could not locate the included file '/cloud/trial-account-limits.md' in any of ["/Users/gregorythrone/Documents/Molecula/GitPull/documentation/_includes", "/Library/Ruby/Gems/2.6.0/gems/jekyll-theme-primer-0.6.0/_includes"]. Ensure it exists in one of those directories and is not a symlink as those are not allowed in safe mode.
```

File: cloud/cloud-tables/_posts/2022-11-14-cloud-table-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Where do I find the table endpoint in FeatureBase Cloud?
+---
+
+The table endpoint is used when importing data from an external data source.
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-14-cloud-table-endpoint.md]: fixed the text and kept the file.
can either delete now and all references, or delete when the functionality is removed

File: cloud/cloud-tables/_posts/2022-11-13-cloud-table-drop.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: How do I drop a table in FeatureBase Cloud?
+---
+
+There are a number of reasons why you may choose to drop a table, including:
+* incorrect configuration
+* incorrect name
+* database deletion
+
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-13-cloud-table-drop.md:10]: added

File: cloud/cloud-tables/_posts/2022-11-10-cloud-table-create.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Step 1: view tables
+
+{% include /cloud/cloud-view-table-list.md %}
+
+## Step 2: create table
+
+1. Click **Create table**
+2. Select the destination database if available.
+3. Enter a name and description if required
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-10-cloud-table-create.md]: reworded to make betterer

File: _includes/concepts/create-table-summary.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+A table is a database object that stores data from an external data source.
+
+FeatureBase tables are similar to those in traditional relational databases:
+* tables with columns
+* columns with data types and constraints.
+
+The first column is automatically created on all tables and serves as the primary key.
+
+When created, a table is assigned a:
+* Table endpoint for use when importing data from a data-source
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply _includes/concepts/create-table-summary.md]: fixed the text

File: _includes/concepts/create-table-summary.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+A table is a database object that stores data from an external data source.
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply _includes/concepts/create-table-summary.md]: fixed

File: _includes/cloud/cloud-pk-table.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+The table ID column is used as the table primary key and must uniquely represent a record in your table.
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply _includes/cloud/cloud-pk-table.md]: fixed

File: cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: Where do I find the table Query Endpoint in FeatureBase Cloud?
```
Comment (gthrone25) [inline cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md:2]: > Where do I find the Query Endpoint in FeatureBase Cloud?

This endpoint is for a database, not a table, so this needs to change to not mislead

File: cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+The table query endpoint is used to query a FeatureBase Cloud table.
```
Comment (gthrone25) [inline cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md:5]: This endpoint is for a database, not a table, so this needs to change to not mislead:

> The query endpoint is used to query tables in a FeatureBase database

File: _includes/concepts/create-table-summary.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+A table is a database object that stores data from external sources.
+
+FeatureBase tables are similar to those in traditional relational databases:
+* tables with columns
+* columns with data types and constraints.
+
+The first column is automatically created on all tables and serves as the primary key.
+
+When created, a table is assigned a Query endpoint for use when querying data using the HTTPS streaming endpoint.
```
Comment (gthrone25) [inline _includes/concepts/create-table-summary.md:9]: Let's get rid of this line. Databases have query endpoints that can be used to query ALL tables in the database.

File: cloud/cloud-tables/_posts/2022-11-18-cloud-table-manage.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+{% include /concepts/create-table-summary.md %}
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Learn how to manage Cloud databases](/cloud/cloud-setup/databases-overview)
+
+## Trial accounts
+
+{% include /cloud/trial-account-limits.md %}
```
Comment (gthrone25) [inline cloud/cloud-tables/_posts/2022-11-18-cloud-table-manage.md:16]: Is this needed for tables? There are no trial limits in terms of number of tables

File: _includes/concepts/data-modeling-overview.md [hunk 11]
```diff
@@ -0,0 +1,3 @@
+Data modeling must be performed before creating database tables to ensure:
+* source column data is matched to FeatureBase data types prior to transformation and import
```
Comment (gthrone25) [inline _includes/concepts/data-modeling-overview.md:2]: > source data is mapped to FeatureBase data types prior to transformation and ingest

**Review by gthrone25 — COMMENTED** [review]
Build works! Added a few more comments now that I can see it all together

File: cloud/cloud-tables/_posts/2022-11-10-cloud-table-create.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
+---
+title: How do I create a table in FeatureBase Cloud?
```
[the remark this answers is not in the document]
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-10-cloud-table-create.md]: fixed this yesterday, sorry didn't mention :D`

File: _includes/concepts/create-table-summary.md (hunk 9 above)
Comment (ljsinclair) [reply _includes/concepts/create-table-summary.md]: sorted. as in removed

File: cloud/cloud-tables/_posts/2022-11-18-cloud-table-manage.md (hunk 10 above)
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-18-cloud-table-manage.md]: removed

File: _includes/concepts/data-modeling-overview.md (hunk 11 above)
Comment (ljsinclair) [reply _includes/concepts/data-modeling-overview.md]: fixed it just like you suggested :)

File: cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md (hunk 7 above)
Comment (ljsinclair) [reply cloud/cloud-tables/_posts/2022-11-15-cloud-table-query-endpoint.md]: Ah, ok. That wasn't obvious from the UI.
Fixed it :)

**Comment by ljsinclair** [comment]
Fixes in :)

**Comment by ljsinclair** [comment]
Hey @gthrone25 am unsure what to do with the above because the files from this branch make changes to the above:
* tables is being deleted because it's replaced by the new stuff
* cloud-streaming-overview now hyperlinks to the new table stuff.

**Comment by ljsinclair** [comment]
These conflicts aren't --
* tables.md is deleted because replaced by new pages
* cloud-streaming-overview.md hyperlinks to the new pages.

**Review by gthrone25 — APPROVED** [review]
LGTM

**Comment by ljsinclair** [comment]
sorted out in `tables-new` branch/pr

**Pull request closed without merging**